src/libraries/System.Data.Common/tests/TrimmingTests/DbConnectionStringBuilder.cs

```diff
@@ -0,0 +1,96 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Data.Common;
+using System.ComponentModel;
+using System.Collections;
+
+class Program
+{
+    static int Main(string[] args)
+    {
+        DbConnectionStringBuilder2 dcsb2 = new();
+        ICustomTypeDescriptor td = dcsb2;
+        if (td.GetClassName() != "DbConnectionStringBuilder2")
```

Comment:
Can you put `DbConnectionStringBuilder2` in a namespace, so we can test that it returns the "FullName" correctly?